librz/analysis/arch/ppc/ppc_il_ops.c

```diff
@@ -255,13 +256,13 @@ static RzILOpEffect *store_op(RZ_BORROW csh handle, RZ_BORROW cs_insn *insn, con
 	case PPC_INS_STWCIX:
 	case PPC_INS_STDCIX:
 #endif
-		base = VARG(rA);
+		base = rA ? VARG(rA) : NULL;
```

Comment:
Is this whole `rA ? VARG() : NULL` thing new in `v5`? And is the assumption, in this case correct, that `rA` will be `NULL` if it is not a register (because the operand value in Capstone is a union).

Reply:
tbh i don't know if it is a bug or else.
From the disassembly prospective, i disassembled these using objdump and it reported the same instruction.
but essentially instead of a register you can have `zero` as base (thus rA is NULL)